src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -21,6 +24,10 @@
     @FXML
     private TextField commandTextField;
 
+    @FXML
```

Comment:
I think you should put `copyButton` in `ResultDisplay` instead of `CommandBox`.

Putting it in `CommandBox` might be confusing to users from a UX perspective (as you would expect it to be a submit button instead)